workshops/09-Polymorphism-Workshop/README.md

```diff
@@ -0,0 +1,151 @@
+# Polymorpism
+
+One fundamental feature of object-oriented programming is _polymorpism_.
```

Comment:
I'd probably try for a one-line summary of what polymorphism is here, by way of introduction - I think it will help contextualise the example that follows.

A bash at it:
> In broad terms, polymorphism is using different concrete components - with the same external interface - to cope with different situations.

Why having the same external interface is important bears mentioning, but probably not as part of this introductory sentence.